jfxname resolution fails on windows

```shell
jbang https://gist.github.com/maxandersen/07ae66cd3e86ae4cf9732b55f8ad5418
[jbang] Resolving dependencies...
[jbang]     Resolving info.picocli:picocli:4.2.0...Done
[jbang]     Resolving org.openjfx:javafx-base:11.0.2:windows...nov 19, 2020 8:45:41 PM org.jboss.shrinkwrap.resolver.impl.maven.logging.LogTransferListener transferFailed
ADVERT╩NCIA: Failed downloading org/openjfx/javafx-base/11.0.2/javafx-base-11.0.2-windows.jar from https://jcenter.bintray.com/. Reason:
org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact org.openjfx:javafx-base:jar:windows:11.0.2 in jcenter (https://jcenter.bintray.com/)
[jbang] [ERROR] Could not resolve dependency
[jbang] Run with --verbose for more details

```

--- a/src/main/java/dev/jbang/Detector.java
+++ b/src/main/java/dev/jbang/Detector.java
@@ -26,6 +26,8 @@ public class Detector extends kr.motd.maven.os.Detector {
 		String os = properties.getProperty("os.detected.name");
 		if (os.equals("osx")) {
 			os = "mac";
+		} else if (os.equals("windows")) {
+			os = "win";
 		}
 		properties.setProperty("os.detected.jfxname", os);
 		System.setProperty("os.detected.jfxname", os);

--- a/src/test/java/dev/jbang/DependencyResolverTest.java
+++ b/src/test/java/dev/jbang/DependencyResolverTest.java
@@ -98,6 +98,25 @@ class DependencyResolverTest {
 	}
 
 	@Test
+	void testResolveJavaFXWithAether() {
+
+		dev.jbang.Detector detector = new dev.jbang.Detector();
+		detector.detect(new Properties(), Collections.emptyList());
+
+		DependencyUtil dr = new DependencyUtil();
+
+		List<String> deps = Arrays.asList(
+				PropertiesValueResolver.replaceProperties("org.openjfx:javafx-base:11.0.2:${os.detected.jfxname}"));
+
+		List<ArtifactInfo> artifacts = dr.resolveDependenciesViaAether(deps,
+				Arrays.asList(toMavenRepo("jcenter")), false,
+				true, true);
+
+		assertEquals(1, artifacts.size());
+
+	}
+
+	@Test
 	void testResolveDependenciesWithAether() {
 
 		DependencyUtil dr = new DependencyUtil();

